src/main/java/com/selfxdsd/todocli/TodoFinderCli.java

```diff
@@ -51,8 +62,51 @@
     public static void main(final String[] args) {
         logger = LoggerFactory.getLogger(TodoFinderCli.class);
 
-        String root = ".";
-        logger.info("Running TodoCLI v1.0 from directory '" + root + "'\n");
+        initOptions(args);
+    }
+
+    /**
+     * Initializes program options, i.e. possible command line arguments.
+     *
+     * @param args Command line arguments
+     */
+    private static void initOptions(final String[] args) {
```

Comment:
@lukacupic Logic in this method should probably be pulled out in a proper object which will be an Iterable of String (each option being a String). 

But we can do this in a future task. Then, we will also be able to write unit tests for it.

Reply:
Yes, as more options are added, a proper object should replace the method. Feel free to open this up as an issue if you already have some options in mind.